internal/web/info_controller.go

```diff
@@ -0,0 +1,39 @@
+package web
+
+import (
+	"net/http"
+
+	"github.com/suse/carrier/internal/cli/clients"
+	"github.com/suse/carrier/version"
+)
+
+type InfoController struct {
+}
+
+func (hc InfoController) Index(w http.ResponseWriter, r *http.Request) {
+	// TODO: Second return value is always nil (the cleanup function)?
+	client, _, err := clients.NewCarrierClient(nil)
```

Comment:
I think this logic of finding information about version, platform should be in the internal/api/v1/info.go just like how logic is in internal/api/v1/applications.go for fetching applications info. Also, I think internal/api/v1/version.go should return the version info.

Reply:
True, I just got lazy and avoided vuejs just for the info page. But you are right, we should be consistent. Do it in a new PR?